### DISCUSSION TITLE
Snap Package and merges

### DISCUSSION THREAD

**Discussion by thecodejedi** [properlypurple/braus#35] [General]
Hey @properlypurple ,

i created a snap package for braus and also merged the pending pull requests.
The projects seems to be "abandoned" and I would like to take it over.

Are you interested in working togehter on it? @properlypurple I'm happy to share access to the snapstore and everything that is needed :)

https://snapcraft.io/braus

**Comment by properlypurple** [comment]
@thecodejedi Please feel free to fork, rename, and publish on Snap or any other place :)
